src/floki_selector_lexer.xrl

```diff
@@ -22,6 +22,7 @@ Rules.
 \({ODD}\)                            : {token, {pseudo_class_odd, TokenLine}}.
 \({EVEN}\)                           : {token, {pseudo_class_even, TokenLine}}.
 \({PSEUDO_PATT}\)                    : {token, {pseudo_class_pattern, TokenLine, remove_wrapper(TokenChars)}}.
+\({QUOTED}\)                         : {token, {pseudo_class_quoted, TokenLine, remove_wrapper(remove_wrapper(TokenChars))}}.
```

Comment:
Because {QUOTED} will capture from the first quotation mark to the next, something like `("searching for this (and that)")` should be fine (though I didn't test to make sure).